src/routes/docs/products/functions/one-click-deployment/+page.markdoc

```diff
@@ -0,0 +1,123 @@
+---
+layout: article
+title: One-Click Deployment
+description: Make your function available to deploy with one click for other users.
+---
+
+Appwrite allows you to make your function available to deploy with one click for other users. This is useful if your function is a template or an open-source project. Using one-click deployment link will ensure your function is easy to setup and deploy for other users, with you providing all the necessary information and configuration.
+
+You may add the deploy link to your GitHub repository's README file or any other documentation you may have.
+
+# The deploy button {% #the-deploy-button %}
+
+You may use the deploy button wherever you want to display a button to deploy your function with Appwrite. The button is hosted on the following URL:
```

Comment:
let's show what the button looks like when embedded (both light and dark mode) like on the assets page